simde/x86/avx.h

```diff
@@ -2797,6 +2797,48 @@ simde_mm256_cmp_ps (simde__m256 a, simde__m256 b, const int imm8)
   #define _mm256_cmp_ps(a, b, imm8) simde_mm256_cmp_ps(a, b, imm8)
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m256
+simde_x_mm256_copysign_ps(simde__m256 dest, simde__m256 src) {
+  simde__m256_private
+    r_,
+    dest_ = simde__m256_to_private(dest),
+    src_ = simde__m256_to_private(src);
+
+  #if defined(simde_math_copysignf)
+    SIMDE_VECTORIZE
+    for (size_t i = 0 ; i < (sizeof(r_.f32) / sizeof(r_.f32[0])) ; i++) {
+      r_.f32[i] = simde_math_copysignf(dest_.f32[i], src_.f32[i]);
+    }
+  #else
+    simde__m256 sgnbit = simde_mm256_xor_ps(simde_mm256_set1_ps(SIMDE_FLOAT32_C(0.0)), simde_mm256_set1_ps(-SIMDE_FLOAT32_C(0.0)));
```

Comment:
This is such an ugly hack, I hate to see it propogated ☹.  Luckily it shouldn't be reached very often, since `simde_math_copysignf` will almost certainly be defined…

Reply:
Yes in most of the cases it would not matter, but I don't see any other workaround for this.